lib/diagnostic.js

```diff
@@ -8,21 +8,34 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let end = arguments[0];
+  for (let i = 1; i < arguments.length; i++) {
+    if (arguments[i] < end) {
+      end = arguments[i];
+    }
+  }
+  console.log(end);
 };
 
```

Comment:
```js
const min = function min() {
  let result = arguments[0];
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }

  return result;
};
```

this is really close, i think using different naming conventions confused you a bit. This is technically right but you want to use const (this shouldn't change) and use conventions like result instead of end. It's right, I just wanted you to know the proper naming convention.